### PR TITLE
deprecated replacements section removed

### LINKED ISSUE #187: chore: Handle deprecation notices for goreleaser
**Is this a BUG REPORT or FEATURE REQUEST?**:
CHORE
**What happened**:
```
• DEPRECATED: `archives.replacements` should not be used anymore, check https://goreleaser.com/deprecations#archivesreplacements for more info
```

**What you expected to happen**:
Cleanup [.goreleaser.yml](https://github.com/keikoproj/addon-manager/blob/af4c63c20fec4ee612ab0b437dc95f81ab6aa9af/.goreleaser.yml#L60-L65) to address Goreleaser v4 deprecrations
https://goreleaser.com/deprecations/#archivesreplacements

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Addon Manager version
- Kubernetes version :
```
$ kubectl version -o yaml
```

**Other debugging information (if applicable)**:
- Addon status:
```
$ kubectl describe addon <addon-name>
```
- controller logs:
```
$ kubectl logs <addon-manager-pod>
```

### PR DESCRIPTION
**Pull request by anushkabishnoi**
Fixes #187

### CODE REVIEW

**Comment by kevdowney** [comment]
Hi @anushkabishnoi Thanks for this PR, you'll need to sign the DCO and use signed commits, so this PR will need to be resubmitted.

File: .goreleaser.yml [hunk 1]
```diff
@@ -57,12 +57,16 @@ archives:
...
       - addonctl
     name_template: "{{ .ProjectName }}_{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
-    replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+
+archives:
```
Comment (kevdowney) [inline .goreleaser.yml:61]: Hey so we don't need this section at all, I'll comment above what we need todo.

File: .goreleaser.yml [hunk 2]
```diff
@@ -57,12 +57,16 @@ archives:
     builds:
       - addonctl
     name_template: "{{ .ProjectName }}_{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
```
Comment (kevdowney) [inline .goreleaser.yml:59]:
```suggestion
    name_template: "{{ .ProjectName }}_{{ .Binary }}_{{ .Version }}_{{ title .Os }}_{{- if eq .Arch "amd64" }}x86_64
      {{- else if eq .Arch "386" }}i386
      {{- else }}{{ .Arch }}{{ end }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
```

**Review by kevdowney — CHANGES_REQUESTED** [review]

**Comment by anushkabishnoi** [comment]
> Hi @anushkabishnoi Thanks for this PR, you'll need to sign the [DCO](https://github.com/keikoproj/addon-manager/pull/189/checks?check_run_id=10717472466) and use signed commits, so this PR will need to be resubmitted.

Okay so do I close this one, and then create another pr, via CLI with the sign-off, right?

**Pull request closed without merging**